recipes/bezier/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "bezier" %}
+{% set version = "0.2.1" %}
```

Comment:
`0.2.1` was the latest release as of January 1 (that was when the [build failed][1] for this PR).

I'm happy to close this PR / modify this one for the `0.3.0` release. I may also make a `0.4.0` release today, so we could wait for that. (Let me know what you think, I don't want to be a burden.)

[1]: https://travis-ci.org/conda-forge/staged-recipes/builds/188162676